scripts/sct_create_mask.py

```diff
@@ -104,37 +103,29 @@ def create_mask(param):
 
     # Get output folder and file name
     if param.fname_out == '':
-        param.fname_out = param.file_prefix + file_data + ext_data
+        param.fname_out = os.path.abspath(param.file_prefix + file_data + ext_data)
 
     path_tmp = sct.tmp_create(basename="create_mask", verbose=param.verbose)
 
     sct.printv('\nCheck orientation...', param.verbose)
-    orientation_input = get_orientation(Image(param.fname_data))
+    orientation_input = Image(param.fname_data).orientation
     sct.printv('.. ' + orientation_input, param.verbose)
     reorient_coordinates = False
 
     # copy input data to tmp folder
-    convert(param.fname_data, os.path.join(path_tmp, "data.nii"))
+    Image(param.fname_data).change_orientation("RPI").save(os.path.join(path_tmp, "data_RPI.nii"))
     if method_type == 'centerline':
-        convert(method_val, os.path.join(path_tmp, "centerline.nii.gz"))
+        Image(method_val).change_orientation("RPI").save(os.path.join(path_tmp, "centerline_RPI.nii"))
```

Comment:
any reason to choose nii over nii.gz? i usually prefer nii.gz for sparse data (centerline, seg) because it compresses amazingly well and for extremely large matrix size it speeds up i/o quite a bit

Reply:
Only for temporary objects.

Here's the rationale:

Today a typical computer
- performs gzip compression at 100 MB/s
- writes to mass storage at > 300 MB/s (ssd) or > 150 MB/s (hdd)
- reads from mass storage at > 500 MB/s (ssd) or > 150 MB/s (hdd)

So using .nii makes the most sense from a speed perspective.

Also, arrays from nii are memory-mapped (whereas those from nii.gz are obviously not).
```python
>>> img1 = nibabel.load("pouet.nii.gz")
>>> type(img1.get_data())
<class 'numpy.ndarray'>
>>> img0 = nibabel.load("pouet.nii")
>>> type(img0.get_data())
<class 'numpy.core.memmap.memmap'>
```

Reply:
cool, i'm convinced. Only for temporary objects though. Storage is still $$$